server/tests/controllers/goal_controller/test_goal.py

```diff
@@ -0,0 +1,94 @@
+"""Tests for the goal CRUD controllers."""
```

Comment:
those are the new tests, other changes in tests  it's just splitting from single module to package